### PR TITLE
Added dropdown component

### PR DESCRIPTION
**Pull request by amaldinesh7**
@goutham-subramanyam _a

Please fix the UI issues.

### CODE REVIEW

**Comment by amaldinesh7** [comment]
@edwinbbu _a Please review.

**Comment by amaldinesh7** [comment]
Fixes #361

File: package.json [hunk 1]
```diff
@@ -73,20 +73,24 @@
   },
   "dependencies": {
     "@bigbinary/neeto-icons": "^1.4.0",
+    "@popperjs/core": "^2.9.2",
```
Comment (edwinbbu) [inline package.json:76]: @amaldinesh7 Are we using this package. I don't find an import. Can you remove it?

File: package.json [hunk 2]
```diff
@@ -73,20 +73,24 @@
...
     "postcss": "^8.2.9",
     "prop-types": "^15.7.2",
     "raw-loader": "^4.0.2",
     "react-color": "^2.19.3",
     "react-hotkeys-hook": "^3.3.2",
     "react-linkify": "^1.0.0-alpha",
-    "react-router-nav-prompt": "^0.4.1"
+    "react-popper": "^2.2.5",
+    "react-router-nav-prompt": "^0.4.1",
+    "yup": "^0.30.0"
```
Comment (edwinbbu) [inline package.json:93]: Remove `yup` this package. It is not used.

File: example/v2/Dropdowns.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import React, { useState } from "react";
+import { Dropdown } from "../../lib/v2";
+import Header from "./Header";
+
+const Dropdowns = () => {
+  // const [loading, setLoading] = useState(false);
+  const [dropdownOne, setDropdownOne] = useState(false);
+  // const [dropdownTwo, setDropdownTwo] = useState(false);
+  // const [dropdownThree, setDropdownThree] = useState(false);
```
Comment (edwinbbu) [inline example/v2/Dropdowns.js:9]: Can you remove the comments.

**Review by edwinbbu — CHANGES_REQUESTED** [review]
@amaldinesh7 _a

File: package.json (hunk 1 above)
Comment (amaldinesh7) [reply package.json:76]: It is a dependency package of react-popper. @edwinbbu

**Comment by amaldinesh7** [comment]
@edwinbbu _a Comments fixed

**Pull request merged**